src/resultsListController.ts

```diff
@@ -480,26 +484,29 @@ export class ResultsListController implements Disposable {
                     valueB = a[this.sortBy.column];
                 }
 
-                if (valueA === undefined || valueA.value === undefined) {
-                    comp = -1;
-                    if (valueB === undefined || valueB.value === undefined) {
+                if (valueA === undefined ||
```

Comment:
Pull this into an equality function.

Reply:
Doing so would remove the TypeScript compiler's ability to know they are not undefined on line 498.

I've made this code easier to understand though.

---
In reply to: [408282753](https://github.com/microsoft/sarif-vscode-extension/pull/243#discussion_r408282753) [](ancestors = 408282753)